### DISCUSSION TITLE
Get SPD data

### DISCUSSION THREAD

**Discussion by PeterVaccaITR** [memtest86plus/memtest86plus#303] [Q&A] [answered]
Hello,
 I am trying to figure out how to get SPD data using memtest. I can successfully boot from USB or PXE and manually chose which tests to run, but I don't see any option about displaying the SPD infomration.

**Comment by x86fr** [comment]
SPD Data (if supported by your motherboard) is displayed on main screen.

What are you looking for regarding SPD Data exactly?

**Reply by PeterVaccaITR** [reply]
<img width="564" alt="Screenshot 2023-05-05 at 4 25 51 PM" src="https://user-images.githubusercontent.com/83974438/236562629-62c63e34-8449-4cb9-a6b8-cec62cf1f49e.png">

Is what I currently see. Is this the main screen?

I am looking to grab the complete raw SPD data if possible, short of that, the first thing I am looking to find is manufacturing date.

**Reply by x86fr** [reply, marked as answer]
SPD reading is currently not supported on  many high-end server motherboards. SMBus architecture on these boards is usually non-standard and often requires specific quirks for each board. 

Adding support for a specific server board is not extremely difficult (unless the board manufacturer really doesn't want to), but we must have access to the hardware.

**Reply by PeterVaccaITR** [reply]
We could provide that. Physical or remote with BMC/SOL access?

**Reply by PeterVaccaITR** [reply]
Hi @x86fr 
 If we can provide either a physical node (it requires a BUS bar for power) or remote access, would it be possible to figure out the quirks for this board?

**Reply by x86fr** [reply]
Yes, but I'm working on other part of the code right now. Remote access can be enough if there is a simple way to boot .efi binary

**Reply by PeterVaccaITR** [reply]
Understood, will put that environment together. What is the best way to provide the details and credentials?
